packages/libfabric/package.py

```diff
@@ -74,6 +76,9 @@ def configure_args(self):
         if '+cuda' in spec:
             config_args.append('--with-cuda={0}'.format(self.spec['cuda'].prefix))
 
+        if '+enable-cuda-dlopen' in spec:
```

Comment:
are the `cuda` and `enable-cuda-dlopen` variants independent, or do you need to have the former to then also use the latter?  Just checking if we need some logic to enforce that or not.